mathesar_ui/src/App.svelte

```diff
@@ -11,6 +15,21 @@
   import { setNewClipboardHandlerStoreInContext } from './stores/clipboard';
   import { modal } from './stores/modal';
   import ModalRecordSelector from './systems/record-selector/ModalRecordSelector.svelte';
+  import { loadLocaleAsync } from './i18n/i18n-load';
+  import { setLocale } from './i18n/i18n-svelte';
+
+  /**
+   * Later the translations file will be loaded
+   * in parallel to the FE's first chunk
+   */
+  let isTranslationsLoaded = false;
+  (() => {
+    void loadLocaleAsync('en').then(() => {
+      setLocale('en');
+      isTranslationsLoaded = true;
+      return true;
+    });
+  })();
```

Comment:
Why the IIFE here? Should the go into an `onMount` function instead? Why `return true`? Why `.then` instead of `await`?

Reply:
- Re-wrote this to use async/await
- Not doing this in `onMount` since do not want to wait to fetch the file until the first render. 
- Not returning `true` anymore
- Anyways, in [this](https://github.com/centerofci/mathesar/pull/3102) PR this whole piece of code is going away.

Reply:
Here: 8e8f6c5